src/Standards/ISAAC/ruleset.xml

```diff
@@ -56,6 +56,8 @@
                 <element key="empty" value="null"/>
                 <element key="isset" value="null"/>
                 <element key="is_null" value="null"/>
+                <element key="serialize" value="null"/>
```

Comment:
Please let me know if this suffices or that we want to replace `value="null"` with `value="json_encode"` (same for unserialize) to suggest an alternative?